hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,17 @@
+# See bottom of the script for the command that kicks off the script
+require 'English'
+
+def reset
+  stop_services
+  reset_pulp
+end
```

Comment:
I think this is small enough that it can be inlined instead of a separate method.